governance/votes/Executive vote - November 5, 2021.md

```diff
@@ -0,0 +1,98 @@
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- [Fee In (tin)](https://makerdao.world/en/learn/governance/module-psm) and [Fee Out (tout)](https://makerdao.world/en/learn/governance/module-psm) parameters for PSM-USDC-A and PSM-USDP-A will be lowered to 0%.
+- Multiple changes to [Stability Fees](https://makerdao.world/en/learn/governance/param-stability-fee), [Maximum Debt Ceilings (line)](https://makerdao.world/en/learn/governance/module-dciam), and [Target Available Debt (gap)](https://makerdao.world/en/learn/governance/module-dciam) as proposed by the [MakerDAO Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355) and detailed below.
+- 359,120.80 DAI will be distributed to the Development and UX (DUX-001) Core Unit Auditors Wallet.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Lower PSM Fees
+
+As per [this poll](https://vote.makerdao.com/polling/QmSkYED5?network=mainnet#poll-detail), the PSM Fees will be lowered as detailed below if this executive proposal passes.
+
+- Lower the PSM-USDP-A [Fee In (tin)](https://makerdao.world/en/learn/governance/module-psm) parameter from 0.10% to 0.0%.
+- Lower the PSM-USDC-A [Fee In (tin)](https://makerdao.world/en/learn/governance/module-psm) parameter from 0.20% to 0.0%.
+
+For more information on this proposal, please see [this signal request](https://forum.makerdao.com/t/signal-request-set-psm-fees-to-0/10894).
+
+### MOMC Parameter Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmXDCCPH?network=mainnet#poll-detail), the following parameter changes will be made if this executive proposal passes.
```

Comment:
Can we say something like this poll on the October proposal from the MOMC. Also link MOMC to the forum post where we introduce it and say what it is.